test/Microsoft.Extensions.Logging.EventSource.Test/EventSourceLoggerTest.cs

```diff
@@ -511,6 +556,12 @@ protected override void OnEventSourceCreated(System.Diagnostics.Tracing.EventSou
                 {
                     _loggingEventSource = eventSource;
                 }
+
+                if (_enableWhenCreated != null)
```

Comment:
Is this guaranteed to be called sequentially? What if the event source is being created while `EnableEvents` is being called?

Reply:
It's testing infrastructure, we only call them sequentially.

Reply:
Ah, I see, this is the test listener.